worlds/alttp/Options.py

```diff
@@ -426,12 +427,6 @@ class BeemizerTrapChance(BeemizerRange):
     display_name = "Beemizer Trap Chance"
 
 
-class AllowCollect(Toggle):
-    """Allows for !collect / co-op to auto-open chests containing items for other players.
-    Off by default, because it currently crashes on real hardware."""
-    display_name = "Allow Collection of checks for other players"
```

Comment:
Removing this also removes the comment explaining that this is to prevent crashing on real hardware.

Reply:
Since collect no longer crashes on real hardware, as of current releases, yeah, its fine for that comment to go away.